Homework3/Task3.py

```diff
@@ -0,0 +1,23 @@
+# Задайте список из вещественных чисел. 
+# Напишите программу, которая найдёт разницу между максимальным 
+# и минимальным значением дробной части элементов.
+
+#  Пример:
+# [1.1, 1.2, 3.1, 5, 10.01] => 0.2
+
+# как найти дробную часть: разделить число с дробной частью на 1 или
+# через вычитание
+
+# import math
+# lst = [1.1, 1.2, 3.1, 5, 10.01] 
+# for i in lst:
+#     print(math.modf(i))
+
+lst = [1.17, 1.28, 3.124, 5.7, 10.52]
+lst2 = []
+res = 0
+for n in lst:
+    res = (round((n - n//1), 4))
```

Comment:
```suggestion
    res = round((n - n // 1), 4)
```
Кстати, запись `n - n // 1` эквивалентна  `n % 1`